client/package.json

```diff
@@ -8,10 +8,13 @@
     "nodemailer": "^6.3.1",
     "react": "^16.10.2",
     "react-dom": "^16.10.2",
+    "react-redux": "^7.1.3",
     "react-router-dom": "^5.1.2",
     "react-scripts": "3.2.0",
     "react-transition-group": "^4.3.0",
     "reactstrap": "^8.1.1",
+    "redux": "^4.0.4",
+    "redux-thunk": "^2.3.0",
     "uuid": "^3.3.3"
   },
   "scripts": {
```

Comment:
Uninstall the redux dependencies.

Reply:
Deleted

Reply:
UPDATE: Uninstalled through bash as you ill advised to manually deleting them.